docs/custom/index.md

```diff
@@ -27,6 +27,8 @@ keywords: keyword1,keyword2
 
 # enable presenter mode, can be boolean, 'dev' or 'build'
 presenter: true
+# enable exporting UI, can be boolean, 'dev' or 'build'
+exportingUI: dev
```

Comment:
I think just `export` would be good enough :)

Reply:
We already have an `export` field in the headmatter:

```yaml
# export options
# use export CLI options in camelCase format
# Learn more: https://sli.dev/guide/exporting.html
export:
  format: pdf
  timeout: 30000
  dark: false
  withClicks: false
  withToc: false
```

Reply:
How about `export.browser`?

Reply:
I guess that would be fine. We could update the docs mentioning about that. The name `exportingUI` feels a bit verbose and we don't usually have the concept of `UI`. Or I don't know if we could call it `exporter` to differentiate with the CLI export

Reply:
Or `browserExport` or `browserExporter` to be more specific?